hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -2498,7 +2498,7 @@ case class TableKeyByAndAggregate(
   expr: IR,
   newKey: IR,
   nPartitions: Option[Int] = None,
-  bufferSize: Int = 50) extends TableIR {
+  bufferSize: Int = -1) extends TableIR {
```

Comment:
just take out the default value?

Reply:
didn't know you could _do_ that in scala. Have default parameters and non-default parameters after them in a parameter list.